src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/ReportPageTests.kt

```diff
@@ -287,6 +296,46 @@ class ReportPageTests : SeleniumTest()
         assertThat(listItems[1].findElements(By.cssSelector("span.remove-reader")).count()).isEqualTo(0)
     }
 
+    @Test
+    fun `can add report readers through user group`()
+    {
+        startApp("auth.provider=montagu")
+
+        addUserWithPermissions(listOf(
+                ReifiedPermission("reports.read", Scope.Global()),
+                ReifiedPermission("users.manage", Scope.Global())
+        ))
+
+        addUserWithPermissions(listOf(), "no.perms@example.com")
+        addUserGroupWithPermissions("test-group", listOf("no.perms@example.com"), listOf())
+
+        insertReport("testreport", "20170103-143015-1234abcd")
+
+        loginWithMontagu()
+        driver.get(RequestHelper.webBaseUrl + "/reports/testreport/20170103-143015-1234abcd")
+
+        //let existing readers load first
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.cssSelector("#reportReadersListVueApp li")))
```

Comment:
Want to add a check that at this point there is only 1 user?

Reply:
That's really tested in the above test, 'can view report readers', but it probably is worth putting in here too just as a sanity test that adding the user makes a difference